### DISCUSSION TITLE
Problem with form inside Stepper and react-hook-form

### DISCUSSION THREAD

**Discussion by riskezwn** [mantinedev/mantine#3225] [Q&A]
### What package has an issue

@mantine/core

### Describe the bug

Hi all. Trying to create a Stepper component with several fields inside each step, using react-hook-form, I realized that when going from one step to another, the value of the input that was in the same position in the previous step is kept (see codesandbox example)

### What version of @mantine/hooks page do you have in package.json?

5.9.4

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/frosty-morning-cjih4c

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
From our side, we provide a form library and an example of how to work with stepper – https://mantine.dev/form/recipes/#form-with-multiple-steps

If you need help with react-hook-form library, you should contact maintainers or community of this library instead.

**Comment by joachimdalen** [comment]
A bit late, but for anyone that might come across this later on:
I had the same issue using the stepper and the Mantine form library. It seems that if the default value of the form field is `undefined` it will retain the value in input fields when changing step. Setting the default values to an empty string solved the problem.
